docs/run.md

```diff
@@ -334,6 +333,54 @@ uonos@uonos-S210-X12RS-V2:~$ cat /etc/hosts | grep envoy
 uonos@uonos-S210-X12RS-V2:~$
 ```
 
+#### Gui access for Development - KinD
+> The following procedure is for Kubernetes in Docker only.
+
+To access the GUI through KinD a port forwarding connection must be established:
```

Comment:
Perhaps the command for port-forwarding also can be added to make it comprehensive in case a user does not know how to do that.

Reply:
updated - my bad